futures-util/src/try_future/mod.rs

```diff
@@ -193,39 +220,43 @@ pub trait TryFutureExt: TryFuture {
         ErrInto::new(self)
     }
 
-    /// Execute another future after this one has resolved successfully.
-    ///
-    /// This function can be used to chain two futures together and ensure that
-    /// the final future isn't resolved until both have finished. The closure
-    /// provided is yielded the successful result of this future and returns
-    /// another value which can be converted into a future.
-    ///
-    /// Note that because `Result` implements the `IntoFuture` trait this method
-    /// can also be useful for chaining fallible and serial computations onto
-    /// the end of one future.
+    /// Chains on a computation returning another future with this future's
```

Comment:
This phrasing seems confusing to me-- I actually like previous description, and I have an easier time understanding the behavior of the method and the resulting future.

Reply:
Changed it to:

> Executes another future after this one resolves successfully. The success value is passed to a closure to create this subsequent future.

That's close to the original phrasing but it also mentions the closure which is IMO crucial.